strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/NugetHierarchicalLayoutProvider.java

```diff
@@ -61,15 +63,41 @@ public NugetHierarchicalArtifactCoordinates getArtifactCoordinates(String path)
     @Override
     public ArtifactInputStream getInputStream(String storageId,
                                               String repositoryId,
-                                              String path) throws IOException, NoSuchAlgorithmException, ArtifactTransportException
+                                              String path)
+                                                           throws IOException,
+                                                           NoSuchAlgorithmException,
+                                                           ArtifactTransportException
```

Comment:
The code style for this across the project is:

    @Override
    public ArtifactInputStream getInputStream(String storageId,
                                              String repositoryId,
                                              String path)
                throws IOException,
                       NoSuchAlgorithmException,
                       ArtifactTransportException